src/client/components/AchievementComponent/AchieveComponent.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './AchieveComponent.styles.css';
+import keyImage from '../../assets/images/icons/kye-icon.svg';
+import xImage from '../../assets/images/icons/cross-icon.svg';
+import timerImage from '../../assets/images/icons/timer-icon.svg';
+import cupImage from '../../assets/images/icons/kup-icon.svg';
+
+export default function Achievement({ achieveTask, Ranking, timeSpent }) {
+  const minutes = Math.floor(timeSpent / 1);
```

Comment:
what yo mean when you devid number on 1 ??? that dosent make sence...

Reply:
I have changed the code here depends on @priyankapandey13 comment https://github.com/HackYourFuture-CPH/fp-class12/pull/71#discussion_r435948032 so can man reflect the number of minutes directly on the screen instead of seconds.

Reply:
I think @myshuker wants to say that if you divide any number with 1 it will give you the same result with or without, so it is okay to remove this part "/1" 🙂